wiki/addons/third-party/kubejs-irons-spells/page.kubedoc

```diff
@@ -101,9 +100,63 @@ StartupEvents.registry("irons_spellbooks:schools", event => {
 		.setFocus("kubejs:test_focus") // The focus needs to be an item tag. You can add this same tag to an item to make it the focus.
 		.setPowerAttribute(global.attributes.TEST_SPELL_POWER.get()) // This sets the power attribute. We use `.get()` on the attribute to obtain the registry object.
 		.setResistanceAttribute(global.attributes.TEST_SPELL_RESISTANCE.get()) // This sets the resistance attribute.
+		// .setPowerAttribute("kubejs:test_spell_power") // You can also use strings or ResourceLocations instead of a direct registry object.
+		// .setResistanceAttribute("kubejs:test_spell_resistance")
 		.setDefaultCastSound('minecraft:entity.chicken.death') // You can also set a default cast sound for each spell in the school.
 		// In 1.20.1 and up, creating spell schools requires a damage type to be inputted.
 		// You can create a damage type using datapacks. https://minecraft.wiki/w/Damage_type
 		// .setDamageType('kubejs:test_spell_damage_type')
 })
 ```
+
+# Custom Spell Books, Staves, and Magic Swords
+```js
+// These all go in startup scripts
+StartupEvents.registry("item", event => {
+	// Other item methods also work on these builders. It is recommended to give the spell books the proper Curios item tag for them to work properly.
+	// (that being "curios:spellbook")
```

Comment:
I already added implicitly the tag of curios:spellbook for all spellbooks.
If it was not supposed to be intended we need to remove from code